app/src/main/java/helper/Gestures.java

```diff
@@ -0,0 +1,29 @@
+package helper;
+
+import android.app.Activity;
+import android.content.Context;
+import android.view.GestureDetector;
+import android.view.MotionEvent;
+
+/**
+ * Created by Sukriti on 7/27/16.
+ */
+public class Gestures extends GestureDetector.SimpleOnGestureListener {
+    //handle 'swipe right' gesture
+    private Context context;
+
+    public Gestures(Context context){
+        this.context = context;
+    }
+
+    @Override
+    public boolean onFling(MotionEvent event1, MotionEvent event2,
+                           float velocityX, float velocityY) {
+
+        if(event2.getX() > event1.getX()) {
+            ((Activity) context).finish();
```

Comment:
It doesn't seem right that the Gestures class could potentially finish the current activity. We should instead have an interface that allows the the implementer to handle a fling, and possibly finish there.